.gitignore

```diff
@@ -0,0 +1,9 @@
+MIT.md
+Reactangle 4.svg
+Reactangle 6 .svg
+node_modules
+<<<<<<< HEAD
+.vscode
```

Comment:
`.vscode` can still be seen in the project directory. Please remember to add all unnecessary files and folders to `.gitignore` so that the project directory looks clean and efficient.